### ISSUE TITLE
v6.25.3 raspbian repo changed 'Suite' from testing to 'stable'

### ISSUE THREAD

**Issue by wozeparrot** [MichaIng/DietPi#2970]
#### Details:
- Date           | Mon  8 Jul 02:33:49 BST 2019
- Bug report     | N/A
- DietPi version | v6.25.3 (MichaIng/master)
- Img creator    | DietPi Core Team
- Pre-image      | Raspbian Lite
- SBC device     | RPi 2 Model B (armv7l) (index=2)
- Kernel version | #896 SMP Thu Jun 20 16:11:44 BST 2019
- Distro         | buster (index=5)
- Command        | G_AGUP
- Exit code      | 100
- Software title | DietPi-Software

#### Steps to reproduce:

1. ...
2. ...

#### Expected behaviour:

- ...

#### Actual behaviour:

- ...

#### Extra details:

- ...

#### Additional logs:
```
Log file contents:
Get:1 http://raspbian.raspberrypi.org/raspbian buster InRelease [15.0 kB]
Hit:2 https://archive.raspberrypi.org/debian buster InRelease
Reading package lists...
E: Repository 'http://raspbian.raspberrypi.org/raspbian buster InRelease' changed its 'Suite' value from 'testing' to 'stable'
```

**Comment by MichaIng** [comment]
@wozeparrot 
Jep this is due to official Debian Buster release on 06/07 (yesterday). I also faced this. You need to run `apt-get update` interactively once and accept this suite change.

**Comment by wozeparrot** [comment]
How would I do that from an install?

**Comment by MichaIng** [comment]
@wozeparrot 
As said, run the following command once from command line/terminal, before starting the dietpi-software install, and confirm if being asked:
```
apt-get update
```
- Impossible to do that from a running install since APT is called without interactive frontend there.
- But you can simply rerun the install afterwards.
________
#### EDIT: Ah one possibility if you face the error prompt is to open a second SSH session or switch to a second TTY. There you can run `apt-get update`, confirm the suite change, and switch back to the install session and hit "`retry`" there.

**Comment by wozeparrot** [comment]
Just an update, running `apt-get update` does not help. You need to run `apt update`.

**Comment by MichaIng** [comment]
@wozeparrot 
Ah `apt-get update` didn't ask for confirming the suite change? Can't remember currently which one I called, apt update should be just a wrapper for apt-get update with prints slightly more output, e.g. how much packages can be updated.

**Comment by wozeparrot** [comment]
No, apt-get update is not interactive, while apt update is.

**Comment by MichaIng** [comment]
`apt-get update` is interactive as well, actually, but perhaps the suite accepting prompt is just part of the `apt update` wrapper. Thanks for this info!

I mark this issue as closed then. Feel free to reopen if required.

**Comment by Trunkzeh** [comment]
For clarity; for those running the experimental Buster image (e.g. the raspberry dietpi buster img) is when you start getting this error is to open a second ssh session and run apt update and then accept the suite change. Then you can close this session and resume the dietpi install :-)

**Comment by MichaIng** [comment]
@Trunkzeh 
Good point. I will also create new Buster images. VirtualBox is already done and Buster the new default/stable.
